src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands_mock.py

```diff
@@ -570,5 +552,55 @@ def test_helm_delete(self, mock_requests_get, mock_get_access_credentials):
             headers=get_authorization_header(EMPTY_GUID, 'password'),
             params=None,
             json=None,
-            data=None,
             verify=mock.ANY)
+
+    @mock.patch('azure.cli.command_modules.acr.helm.get_access_credentials', autospec=True)
+    @mock.patch('requests.request', autospec=True)
+    def test_helm_push(self, mock_requests_get, mock_get_access_credentials):
+        cmd = mock.MagicMock()
+        cmd.cli_ctx = DummyCli()
+
+        response = mock.MagicMock()
+        response.headers = {}
+        response.status_code = 200
+        mock_requests_get.return_value = response
+
+        mock_get_access_credentials.return_value = 'testregistry.azurecr.io', EMPTY_GUID, 'password'
+
+        builtins_open = '__builtin__.open' if sys.version_info[0] < 3 else 'builtins.open'
+
+        # Push a chart
+        with mock.patch(builtins_open) as mock_open:
```

Comment:
what a mock!

One suggestion though I would love to see this test to be split into three so that one failed assertion won't reduce the coverage.